### PR TITLE
add bureaucratprotect for testwiki

### PR DESCRIPTION
**Pull request by Reception123**
I think that we should have bureaucrat protect

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -266,6 +266,11 @@
 			'wikicreator' => array(
 				'createwiki' => true,
 			),
+		'testwiki' => array(
```
Comment (Southparkfan) [inline LocalSettings.php]: "+testwiki"

File: LocalSettings.php [hunk 2]
```diff
@@ -266,6 +266,11 @@
...
+	        	'bureaucrat' => array(
++				'bureaucratprotect' => true,
```
Comment (Southparkfan) [inline LocalSettings.php]: Indent needs to be fixed

File: LocalSettings.php [hunk 3]
```diff
@@ -266,6 +266,11 @@
...
+       			),
```
Comment (Southparkfan) [inline LocalSettings.php]: Fix indent please

**Comment by JohnFLewis** [comment]
New single commit please.

**Comment by Reception123** [comment]
Now #21

**Pull request closed without merging**